core/src/read_write_account_set.rs

```diff
@@ -0,0 +1,354 @@
+use {
+    solana_sdk::{
+        message::{SanitizedMessage, VersionedMessage},
+        pubkey::Pubkey,
+    },
+    std::collections::HashSet,
+};
+
+/// Wrapper struct to check account locks for a batch of transactions.
```

Comment:
just a note that `ReadWriteAccountSet` is similar to `Accounts::AccountLocks` struct, and serves same purpose. Perhaps can remove `Accounts::AccountLocks` after scheduler lands.

Reply:
Realistically, I'll probably be moving AccountLocks out of Accounts and into scheduling. Maybe I should just use it here now - I don't need the read-count which is why I didn't.

Reply:
Another reason I didn't use the AccountLocks is I don't want to add and expose a `clear` interface on that type - the way its' used in accounts/banking stage shouldn't allow for us to clear all locks.